Formula/w/whisper-cpp.rb

```diff
@@ -42,7 +37,13 @@ def caveats
   test do
     cp [pkgshare/"jfk.wav", pkgshare/"for-tests-ggml-tiny.bin", pkgshare/"ggml-metal.metal"], testpath
 
-    system bin/"whisper-cpp", "samples", "-m", "for-tests-ggml-tiny.bin"
+    if OS.mac? && Hardware::CPU.intel?
```

Comment:
Just to clarify, does this mean that the test shouldn't include an if statement?

If that's the case, I think it will only be tested with the --no-gpu option. However, I believe using Metal is almost necessary for practical use cases and I would like to test it as well.